Firmware/mmu2.cpp

```diff
@@ -683,6 +689,8 @@ void MMU2::CheckUserInput(){
             // we'll actually wait for it automagically in manage_response and after it finishes correctly,
             // we'll issue another command (like toolchange)
             logic.ClearPrinterError();
+            lastErrorCode = ErrorCode::OK;
+            lastErrorSource = ErrorSourceNone; // this seems to help clearing the error screen
```

Comment:
These lines should not be needed. LCD blank issue was fixed in https://github.com/prusa3d/Prusa-Firmware/pull/3907

Reply:
I know, that the issue went away in #3907, but I still think cleaning up after this error is a good idea.
I'm considering moving the PrinterError infrastructure from the logic layer into mmu2.cpp.

Reply:
Cleaning the error code is a good point.